lib/travis/addons/serializer/pusher/build.rb

```diff
@@ -118,6 +120,21 @@ def stages
               build.stages
             end
 
+            def created_by
+              sender = build.sender
+
+              if sender
```

Comment:
this could be reduced a little by returning early on the first line, like `return unless sender = build.sender` ... but that's nitpicking :)